src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -215,27 +187,16 @@ public bool ShowUseLevelMenu
         /// <summary>
         /// If UseLevel is enabled on this port.
         /// </summary>
+        [Obsolete("This method will be removed in Dynamo 3.0 - please use the InPortViewModel")]
         public bool UseLevels
         {
             get { return _port.UseLevels; }
         }
 
-        /// <summary>
-        /// Determines whether or not the UseLevelsSpinner is visible on the port.
-        /// </summary>
-        public Visibility UseLevelSpinnerVisible
```

Comment:
Shouldn't this be marked obsolete as it's public?

Reply:
They were not in 2.12 so we can just delete them as 2.13 is not released